pkg/core/options.go

```diff
@@ -76,6 +78,13 @@ func WithMetrics(enabled bool) Option {
 	}
 }
 
+// WithLabel toggles metrics for the core package and its dependencies (e.g. cafs)
```

Comment:
Comment comes from copy/paste

Reply:
noted and changed:  added several comments/godoc strings since this review.